apps/src/sites/studio/pages/levels/_curriculum_reference.js

```diff
@@ -0,0 +1,33 @@
+window.curriculumReference = {
+  onload: onload
+};
+
+/**
+ * Called on load of our iframe. This hides the spinner, shows the iframe, and
+ * sets up a method to keep our iframe correctly sized.
+ */
+function onload() {
+  const iframe = document.getElementById('curriculum-reference');
+  // For some reaosn onload is called multiple times in Chrome. We only want to
+  // start our interval once.
+  if (iframe.style.display !== 'none') {
+    return;
+  }
+
+  // Make iframe visible, hide spinner
+  iframe.style.display = '';
+  document.getElementById('iframe-loading').style.display = 'none';
+
+  // Our iframe can resize for various reasons (initial load, images loaded on
+  // iframe page, window resized). We want to keep the size of the iframe in
+  // sync with the contents of the firame.
+  setInterval(() => {
+    resizeIFrame(iframe);
+  }, 200);
```

Comment:
```js
iframe.contentWindow.addEventListener('load', resizeIFrame, true);
```
Interestingly, this appears to work on IE, but not on Chrome/FF/Safari. In those browsers, the sizes still get out of sync.

I think that using the iframe scrollbar instead would feel somewhat awkward, since you're still going to have some things on the page outside of the scroll area. (like the continue button).